core/domain/stats_jobs_one_off.py

```diff
@@ -231,21 +231,22 @@ def _apply_state_name_changes(cls, prev_stats_dict, change_list):
             dict. A dict representation of an ExplorationStatsModel
                 with updated state_stats_mapping and version.
         """
+        exp_versions_diff = exp_domain.ExplorationVersionsDiff(change_list)
+
         # Handling state additions, deletions and renames.
```

Comment:
Add a note here along the lines of "(Note that the order these are applied is important.)" Also change to say "Handling state deletions, renames and additions (in that order)."

Reply:
Done.